.github/actions/setup-runner/action.yml

```diff
@@ -43,3 +43,21 @@ runs:
     - name: Update /etc/hosts
       shell: bash
       run: echo -e "$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)\t$(hostname -f) $(hostname -s)" | sudo tee -a /etc/hosts
+
+    - name: Setup docker.io mirror
+      shell: bash
+      run: |
+        cat > docker-io-mirror.conf <<!
+        [[registry]]
+        # In Nov. 2020, Docker rate-limits image pulling.  To avoid hitting these
+        # limits while testing, always use the google mirror for qualified and
+        # unqualified `docker.io` images.
+        # Ref: https://cloud.google.com/container-registry/docs/pulling-cached-images
+        prefix="docker.io"
+        location="mirror.gcr.io"
+        !
+        sudo mv docker-io-mirror.conf /etc/containers/registries.conf.d/
+        echo ""
+        echo "Added docker.io mirror via"
+        echo ""
+        cat /etc/containers/registries.conf.d/docker-io-mirror.conf 
```

Comment:
Don't we need to `sudo service (docker|podman) restart` after changing the mirrors?